### PR TITLE
Feature/task 5382 5272 on boarding

### PR DESCRIPTION
**Pull request by KaiTeuber**
## Description
Show flag images and country names as a line layout in OnBoarding screen

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-5382

## Screenshots
Simulator Screen Shot - iPhone 11 - 2021-02-25 at 15 59 43
Simulator Screen Shot - iPhone 11 - 2021-02-25 at 15 59 53
fea

### CODE REVIEW

File: src/xcode/ENA/ENA/Source/Extensions/UIImage+Resize.swift [hunk 1]
```diff
@@ -0,0 +1,30 @@
+////
+// 🦠 Corona-Warn-App
+//
+
+import Foundation
+import UIKit
+
+extension UIImage {
+
+	/*** Resize and create a new UIImage with targetSize and keep the aspect ratio of the original image*/
```
Comment (SwissMountainFrog) [inline src/xcode/ENA/ENA/Source/Extensions/UIImage+Resize.swift:10]: Just wondering, what /*** means ;-) Is this a multiline comment? Shouldn't it be like this?

`// Resize and create a new UIImage with targetSize and keep the aspect ratio of the original`.

File: src/xcode/ENA/ENA/Source/Extensions/Sequence+NSAttributedText.swift [hunk 2]
```diff
@@ -0,0 +1,24 @@
+////
+// 🦠 Corona-Warn-App
+//
+
+import Foundation
+
+/** Helper to join array of NSAttributedString with a separator */
```
Comment (SwissMountainFrog) [inline src/xcode/ENA/ENA/Source/Extensions/Sequence+NSAttributedText.swift:7]: Is, was this a multiline comment? Shouldn't it be changed to single line comment?

**Review by tcfos — APPROVED** [review]

File: src/xcode/ENA/ENA/Source/Scenes/Onboarding/OnboardingInfoViewController.xib [hunk 3]
```diff
@@ -42,7 +42,7 @@
                             <rect key="frame" x="0.0" y="0.0" width="414" height="678.66666666666663"/>
```
Comment (SwissMountainFrog) [inline src/xcode/ENA/ENA/Source/Scenes/Onboarding/OnboardingInfoViewController.xib:42]: You rounded a lot of numbers, shouldn't this `height="678.66666666666663"` not be changed to `679`?

Comment (KaiTeuber) [reply src/xcode/ENA/ENA/Source/Scenes/Onboarding/OnboardingInfoViewController.xib:42]: Xcode did that 👍

**Review by 30mar — APPROVED** [review]

**Pull request merged**